src/types.rs

```diff
@@ -7,6 +7,11 @@ use ethereum_consensus::primitives::{
 };
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
```

Comment:
a bit more compact bc you just imported them up above

```suggestion
#[derive(Serialize, Deserialize, Debug)]
```